fastify.js

```diff
@@ -92,6 +92,16 @@ function build (options) {
     server.on('clientError', handleClientError)
   }
 
+  // JSON body limit option
+  if (options.jsonBodyLimit !== undefined) {
+    if (!Number.isInteger(options.jsonBodyLimit)) {
+      throw new TypeError(`'jsonBodyLimit' option must be an integer. Got: '${options.jsonBodyLimit}'`)
+    }
+    fastify._jsonBodyLimit = options.jsonBodyLimit
+  } else {
+    fastify._jsonBodyLimit = 1000 * 1000 // 1 MB
```

Comment:
Could you use an external constant for this?

Reply:
It’s 1024 * 1024.

Reply:
So we want [`MiB`](https://en.wikipedia.org/wiki/Mebibyte) then? I would actually agree with this since base-2 values are more common in networking applications.

Reply:
I think we should use base2 values.